google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -2512,6 +2516,62 @@ TEST(ConnectionImplTest, CommitSuccessWithStats) {
   EXPECT_EQ(42, commit->commit_stats->mutation_count);
 }
 
+TEST(ConnectionImplTest, CommitSuccessWithoutMaxCommitDelay) {
```

Comment:
We typically don't test the "without" cases, so I'd suggest removing this.

Reply:
Done.